### PR TITLE
Hunterize

### PR DESCRIPTION
**Pull request by AlexChubukov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## RocksDB: A Persistent Key-Value Store for Flash and RAM Storage
+# RocksDB: A Persistent Key-Value Store for Flash and RAM Storage
```
Comment (miasantreble) [inline README.md]: did you mean to change the font of the title?

File: cmake/HunterGate.cmake [hunk 2]
```diff
@@ -0,0 +1,540 @@
+# Copyright (c) 2013-2018, Ruslan Baratov
```
Comment (miasantreble) [inline cmake/HunterGate.cmake]: it this copyright statement intended to be different from the rest of the files?

**Comment by miasantreble** [comment]
do you mind explaining a bit the purpose of this PR?

**Comment by siying** [comment]
@AlexDeveloper24 do you plan to move forward, or should we close it?

**Comment by riversand963** [comment]
Closing due to the following
- the dependency is questionable. https://github.com/hunter-packages/gate no longer accepts public contribution, and I cannot evaluate the risks (both technical and legal)
- no justification of the addition.

**Pull request closed without merging**